specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/examples/GetRoleAssignmentScheduleRequestByScope.json

```diff
@@ -6,59 +6,62 @@
+              "approvalId": null,
+              "scheduleInfo": {
+                "startDateTime": "2020-09-09T21:35:27.91Z",
+                "expiration": {
+                  "type": "AfterDuration",
+                  "endDateTime": null,
+                  "duration": "PT8H"
+                }
               },
-              "roleDefinition": {
-                "id": "/subscriptions/dfa2a084-766f-4003-8ae1-c4aeb893a99f/providers/Microsoft.Authorization/roleDefinitions/c8d4ff99-41c3-41a8-9f60-21dfdad59608",
-                "displayName": "Contributor",
-                "type": "BuiltInRole"
+              "ticketInfo": {
+                "ticketNumber": null,
+                "ticketSystem": null
               },
-              "principal": {
-                "id": "a3bb8764-cb92-4276-9d2a-ca1e895e55ea",
-                "displayName": "User Account",
-                "email": "user@my-tenant.com",
-                "type": "User"
+              "justification": null,
+              "requestorId": "a3bb8764-cb92-4276-9d2a-ca1e895e55ea",
+              "createdOn": "2020-09-09T21:35:27.91Z",
+              "condition": "@Resource[Microsoft.Storage/storageAccounts/blobServices/containers:ContainerName] StringEqualsIgnoreCase 'foo_storage_container'",
+              "conditionVersion": "1.0",
+              "expandedProperties": {
+                "scope": {
+                  "id": "/subscriptions/dfa2a084-766f-4003-8ae1-c4aeb893a99f",
+                  "displayName": "Pay-As-You-Go",
+                  "type": "subscription"
+                },
+                "roleDefinition": {
+                  "id": "/subscriptions/dfa2a084-766f-4003-8ae1-c4aeb893a99f/providers/Microsoft.Authorization/roleDefinitions/c8d4ff99-41c3-41a8-9f60-21dfdad59608",
+                  "displayName": "Contributor",
+                  "type": "BuiltInRole"
+                },
+                "principal": {
+                  "id": "a3bb8764-cb92-4276-9d2a-ca1e895e55ea",
+                  "displayName": "User Account",
+                  "email": "user@my-tenant.com",
+                  "type": "User"
+                }
               }
-            }
-          },
-          "name": "fea7a502-9a96-4806-a26f-eee560e52045",
-          "id": "/providers/Microsoft.Subscription/subscriptions/dfa2a084-766f-4003-8ae1-c4aeb893a99f/providers/Microsoft.Authorization/RoleAssignmentScheduleRequests/fea7a502-9a96-4806-a26f-eee560e52045",
-          "type": "Microsoft.Authorization/RoleAssignmentScheduleRequests"
-        }
-      ]
+            },
+            "name": "fea7a502-9a96-4806-a26f-eee560e52045",
+            "id": "/providers/Microsoft.Subscription/subscriptions/dfa2a084-766f-4003-8ae1-c4aeb893a99f/providers/Microsoft.Authorization/RoleAssignmentScheduleRequests/fea7a502-9a96-4806-a26f-eee560e52045",
+            "type": "Microsoft.Authorization/RoleAssignmentScheduleRequests"
+          }
+        ],
+        "nextLink": "https://getMoreData.net/foo"
```

Comment:
We have added next link in the model but that's only for future compatibility - we do not send it in the response currently that's why I did not have it in the examples. I believe you can remove this and the validations will still pass as they were passing earlier.

Reply:
removed